system/Database/SQLite3/Connection.php

```diff
@@ -56,6 +57,15 @@ class Connection extends BaseConnection
      */
     protected $busyTimeout;
 
+    /**
+     * The setting of the "synchronous" flag
+     *
+     * @var int|null flag
+     *
+     * @see https://www.sqlite.org/pragma.html#pragma_synchronous
+     */
+    protected $synchronous;
```

Comment:
We can type this natively - `?int`

Reply:
I did not add a type for this property simply because other properties are not currently typed. I wanted to adapt to the current state of the class. But you are right. Added.